composer/trainer/trainer.py

```diff
@@ -1144,10 +1145,16 @@ def __init__(
         else:
             eval_metrics = deepcopy(self.state.model.get_metrics(is_train=False))
             model_metric_names = [str(k) for k in eval_metrics.keys()]
+            dataloader_tupled = ensure_tuple(eval_dataloader)
```

Comment:
nit: can we just do this to keep it simple?
```suggestion
            eval_dataloader = ensure_tuple(eval_dataloader)
```